geoh5py/ui_json/utils.py

```diff
@@ -336,7 +336,7 @@ def monitored_directory_copy(
     temp_geoh5 = f"temp{time():.3f}.geoh5"
 
     with fetch_active_workspace(entity.workspace, mode="r"):
-        with Workspace(working_path / temp_geoh5) as w_s:
+        with Workspace().save(working_path / temp_geoh5) as w_s:
```

Comment:
Can we get a .create constructor for this pattern?

Reply:
Good idea. Will propagate the change through.